networking/networking.go

```diff
@@ -47,11 +48,13 @@ type Networking struct {
 
 	hostNS *os.File
 	nets   []activeNet
+
+	netsLoadList map[string]bool
```

Comment:
I'd rather see just a slice of strings or `map[string]struct{}` to better communicate that it's a set.